metabrainz/templates/users/supporters-list.html

```diff
@@ -24,7 +24,7 @@ <h1 class="page-title">Supporters</h1>
         <div class="users-grid">
           {% for user in featured_users %}
             <div class="user">
-              <a href="{{ url_for('users.tier', tier_id=tier.id) }}">
+              {% if user.website_url %}<a href="{{ user.website_url }}">{% endif %}
```

Comment:
I think we should. I doubt many people will click through to the individual tiers page. I think I might toss Zvooq into this bin, so we can see how it looks on the site. Please add this and then feel free to merge.